app/views/submissions/_submission_evaluation.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2>Submission Evaluation</h2>
+<% if @submission.phase.evaluation_form.nil? %>
+  <%= render 'shared/alert_error', alert_heading: t('alerts.no_evaluation_form.heading'), alert_text: t('alerts.no_evaluation_form.text') %>
+<% else %>
+    <%= render partial: 'assigned_evaluators' %>
+    <%= render partial: 'available_evaluators' %>
+<% end %>
```

Comment:
Don't forget the link to "Manage my evaluation panel"
<img width="633" alt="Screenshot 2024-12-20 at 12 01 33 PM" src="https://github.com/user-attachments/assets/f985b405-f3ff-4ede-9cac-f843faf969fa" />